sys/vane/rver.hoon

```diff
@@ -1985,6 +1997,9 @@
         ~|([%bad-channel-wire wire] !!)
     ::
         %timeout
+      ?>  ?=([%b %wake *] sign)
+      =?  moves  ?=(^ error.sign)
+        [[duct %d %flog %crud %wake error.sign] moves]
```

Comment:
Handled this case as per @eglaysher's guidance. Does this seem correct to you @joemfb

Reply:
No, this isn't going to be sufficient. The semantics of getting a `%wake` with an error is that you already got a `%wake` without the error, and something you did crashed. You can't just continue doing the same thing you would normally do (and in fact already did), because it'll just crash again.

The bare minimum is to simply print the error and do nothing else. The objectively correct thing to do is to release/undo any leases/commitments associated with that timer (cancel builds, requests, clean up state, etc.), but I'm not sure what that specifically means in this case.

Reply:
The only thing this handler does is undo leases. If we crashed during this handling, we crashed while trying to free the timer resources on timeout.

So I guess I was wrong, and you should just print, @ixv ?

Reply:
That looks right to me.

Reply:
To be clear, I think @eglaysher is correct and you should only print, not continue with the evaluation in the error case.